.github/workflows/maven-verify.yml

```diff
@@ -30,4 +30,4 @@ jobs:
       jdk-distribution-matrix: '[ "temurin", "zulu", "microsoft", "adopt-openj9" ]'
       jdk-matrix: '[ "8", "11", "17", "18" ]' # "19-ea"
       matrix-exclude: '[{ "jdk": "8", "distribution": "microsoft"},{ "jdk": "19-ea", "distribution": "microsoft"},{ "jdk": "18", "distribution": "microsoft"},{ "jdk": "19-ea", "distribution": "adopt-openj9"},]'
-      maven-matrix: '[ "3.3.9", "3.6.3", "3.8.6" ]'  # Maven versions matrix for verify builds
+      maven-matrix: '[ "3.3.9", "3.6.3", "3.8.6", "3.9.0", "4.0.0-alpha-4" ]'  # Maven versions matrix for verify builds
```

Comment:
Not directly related to this commit (sorry for intruding):
* Should we also build with JDK 19 rather than 18? E.g. use the latest stable JDK?
* And maybe build with 3.8.7 instead of 3.8.6?

Reply:
maybe get rid of 3.6.3?
and agree on removing 18 and replace with 19 (maybe 20-ea if exists?)

Reply:
Yes, I think we should test LTS versions + latest.
I'll use this PR to fix the failing IT too.

Reply:
Though changing the supported JDK may require some additional matrix exclusions, so we may want to tackle that in a different PR, I'll focus on maven versions + fixing the IT for now.